GalnetMonitor/GalnetFeedItemNormalizer.cs

```diff
@@ -67,6 +67,12 @@ private static string Normalize(string value)
                 value = StripHTML(value);
                 value = StripDoubleOrMoreWhiteSpace(RemoveControlChars(value));
                 value = value.Normalize().Trim();
+                if (content)
+                {
+                    int start = value.IndexOf("Body") + 5;
+                    int end = value.LastIndexOf("Date") - start;
```

Comment:
If this feed were available for multiple languages, this might need to be localized.